CONTRIBUTING.md

```diff
@@ -4,25 +4,25 @@
 Welcome to the Prophet community and thank you for your contribution to its continued legacy. 
 We compiled this page with practical instructions and further resources to help you get started.
 
-Please come join us on our [Slack](https://join.slack.com/t/neuralprophet/shared_invite/zt-sgme2rw3-3dCH3YJ_wgg01IXHoYaeCg), you can message any core dev there.
+For an easy start, check out all open issues with the label https://github.com/ourownstory/neural_prophet/labels/good%20first%20issue. 
+They can be done somewhat in isolation from other tasks and will take a couple hours up to a week of work to complete. We appreciate your help!
 
-## Get Started On This
-We created an [Contributing: Issues Overview Page](https://github.com/users/ourownstory/projects/3/views/1) with all tasks where we would appreciate your help! 
-They can be done somewhat in isolation from other tasks and will take a couple hours up to a week of work to complete.
+Please come join us on our [Slack](https://join.slack.com/t/neuralprophet/shared_invite/zt-sgme2rw3-3dCH3YJ_wgg01IXHoYaeCg), you can message any core dev there.
 
 ## Process
-Here's a great [beginner's guide to contributing to a GitHub project](https://akrabat.com/the-beginners-guide-to-contributing-to-a-github-project/#to-sum-up). 
+In summary, follow the steps below to make your first contribution. If you want to have more details on every step, check out this [beginner's guide to contributing to a GitHub project](https://akrabat.com/the-beginners-guide-to-contributing-to-a-github-project/#to-sum-up). 
 
-In Summary: 
-* Fork the project & clone locally.
-* Create an upstream remote and sync your local copy before you branch.
-* Branch for each separate piece of work.
-* Do the work, write good commit messages, and read the CONTRIBUTING file if there is one.
-* Push to your origin repository.
-* Create a new PR in GitHub.
-* Respond to any code review feedback.
+1. Dev install NeuralProphet following [this tutorial](https://github.com/ourownstory/neural_prophet/edit/1166-shorten-contributing.md/CONTRIBUTING.md#dev-install) 
+2. Find an issue to work on by filtering on the label https://github.com/ourownstory/neural_prophet/labels/good%20first%20issue
```

Comment:
It would be great to assign this label somehow or ask people to comment on the issue they wish to take over (otherwise multiple people might end up working on the same issue)

Reply:
done, makes sense!